variables.tf

```diff
@@ -24,8 +24,23 @@ variable "event_types" {
 
 variable "lambda" {
   description = "Map of any additional arguments for the upstream lambda module. See <https://github.com/terraform-aws-modules/terraform-aws-lambda>"
-  type        = any
-  default     = {}
+  type = object({
```

Comment:
There are several attributes here that aren't actually being used in the code. Not sure I've caught all of them, but I'm seeing at least, `build_in_docker`, `memory_size`, `runtime`, and `timeout`.

Reply:
I believe all of the unused attributes have been removed.